servo-tidy.toml

```diff
@@ -5,6 +5,9 @@ check-ordered-json-keys = [
   "./resources/prefs.json",
   "./resources/package-prefs.json",
 ]
+lint-scripts = [
+  "./python/servo/lints/wpt_lint.py",
```

Comment:
I couldn't achieve this without a `RuntimeWarning`, which I guess has something to do with python's import resolution and the absence of `__init__.py` in the subdirectories. Shall we land this for now, and improve it later?